client/app/components/json-view-interactive/json-view-interactive.less

```diff
@@ -3,15 +3,15 @@
 @jvi-gutter: 20px;
 @jvi-spacing: 2px;
 
+.jvi-root {
+  display: block;
+  font-family: @font-family-monospace;
+}
```

Comment:
But without it top-most block will have a different font which is quite noticeable when you expand object/array:
![image](https://user-images.githubusercontent.com/12139186/62032183-b4cbca80-b1f1-11e9-8c8d-961a9e0509a1.png)
![image](https://user-images.githubusercontent.com/12139186/62032111-8c43d080-b1f1-11e9-9ab5-5dc1264db687.png)
(ellipsis on a top level looks differently from nested ones; square braces look differently; icon is a bit unaligned because of wrong font)

Reply:
Oh. Then I guess we'll have to reapply the normal font onto it?

```css

.jvi-root {
  display: block;
  font-family: @font-family-monospace;
}

.jvi-comment {
  font-family: @font-family;
}
```

Reply:
:thinking: I'll try and see how it looks

Reply:
If this is the only issue left in this PR, how about we address it in a separate PR? 

This feature, awesome as it is, is the least important thing in this PR and releasing this PR sooner means less frustrated people who see partial tables...

Reply:
@arikfr It's a one-minute fix, so I'll give it a try. I plan to push all other fixes today, and then we can give another look to it and merge :+1:

Reply:
It looks better when comment has non-monospace font: 
![image](https://user-images.githubusercontent.com/12139186/62143611-fd21e000-b2f8-11e9-9c8d-770826685fcf.png)